### ISSUE TITLE
Manage users and user types

### ISSUE THREAD

**Issue by rdmordido** [leisuretexpainting/leisuretexpainting#4]
Leisuretextpainting must include a function on which it will be able list, create, modify, and remove user details including it's user types such as admistrators, sales, accounts, and staff members.

**Comment by rdmordido** [comment]
@leisuretexpainting 

Please verify what types of users we'll be having into our system.
I saw the following user types from http://dev.leisuretexpainting.com.au/
1. Administrator
2. Contractor
3. Supervisor
4. Quantity Surveyor

**Comment by leisuretexpainting** [comment]
@rdmordido users will be as below and there will different permissions within each type. Would also like functionality to change each users' permissions individually. For example one of our staff is sales so I would like it to automatically select the typical 'sales' permissions but then also have tick boxes to ensure they can then access other sections needed. I know this is an issue for later but thought I would give you an idea now :)
1. Administrator
2. Sales
3. Operations
4. Senior Estimator
5. Estimator
6. Supervisor
7. Staff
8. Contractor

**Comment by rdmordido** [comment]
@leisuretexpainting 

Thank you for the details, I will implement it now with the felexibility for enhancement of managing permissions in the future.

**Comment by rdmordido** [comment]
@leisuretexpainting 

Quick questions here,
Does "contractor" really belongs to our type of users?
Will they be able to login or use our system?

**Comment by leisuretexpainting** [comment]
Yes you can maybe change it to "Subcontractor" we have subcontractor painters who will need to log in to enter timesheets and possibly upload invoices.

**Comment by rdmordido** [comment]
How many sub contractor painters do we have and will log into our system?
And how is it different from the "Administrator"?

**Comment by leisuretexpainting** [comment]
The administrator role will be me and possibly one other person. They will be able to go over all sectors of the system. A sub contractor will be the same as a painter /staff. All they will need to do is enter the times they have worked each day and upload their invoice weekly. At the moment we will have 1 or 2 sub contractor logins but down the line we could have anything from 2-40 subcontractors.
